package.json

```diff
@@ -17,7 +17,8 @@
     "test": "lerna run test --",
     "bootstrap": "lerna bootstrap",
     "release": "lerna version minor --yes && lerna publish from-git --yes",
-    "size": "lerna run size"
+    "size": "lerna run size",
+    "generate": "yarn plop --plopfile generators/plopfile.ts"
```

Comment:
Since the generator is at the monorepo's root, so it's not technically something just related to the UI project, I feel like we need to be more descriptive with the command's name. It's not immediately clear **what** is going to be generated when you run the command at the root. 

Just `generate` would be fine if it was inside `packages/ui`, but since it's here, I suggest we use something like `generate-ui-component`.

Or maybe I'm exaggerating.
😆

Reply:
I liked this suggestion of being more specific about what is being generated!